ROLES.md

```diff
@@ -145,36 +153,35 @@ the area.
 
 Members who frequently contribute code are expected to proactively perform code reviews for the area that they are active in.
 
-## Approver
+## Maintainer
 
-Code approvers are able to both review and approve code contributions. While code review is focused on code quality and correctness,
+Maintainers review and approve code contributions. While code review is focused on code quality and correctness,
 approval is focused on holistic acceptance of a contribution including: backwards / forwards compatibility, adhering to API and flag
-conventions, subtle performance and correctness issues, interactions with other parts of the system, etc. Approver status is scoped to a
+conventions, subtle performance and correctness issues, interactions with other parts of the system, etc. Maintainer status is scoped to a
```

Comment:
Is maintainer status still scoped to a portion of the code base? I might be implicitly, but I don't think it's explicitly called out anymore since there aren't individual OWNERS files pre directory.

Reply:
CODEOWNERS serve the same purpose. I added a sentence about CODEOWNERS and OWNERS files reflecting the maintainer status.